src/containers/org/reducers/deleteExistingOrganizationReducer.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * @flow
+ */
+
+import { Map } from 'immutable';
+import { RequestStates } from 'redux-reqseq';
+import type { SequenceAction } from 'redux-reqseq';
+
+import { IS_OWNER, ORGS, REQUEST_STATE } from '../../../core/redux/constants';
+import { DELETE_EXISTING_ORGANIZATION, deleteExistingOrganization } from '../actions';
+
+export default function reducer(state :Map, action :SequenceAction) {
+  return deleteExistingOrganization.reducer(state, action, {
+    REQUEST: () => state
+      .setIn([DELETE_EXISTING_ORGANIZATION, REQUEST_STATE], RequestStates.PENDING)
+      .setIn([DELETE_EXISTING_ORGANIZATION, action.id], action),
+    SUCCESS: () => {
+      const orgId = action.value.id;
```

Comment:
When you make the change to `deleteExistingOrganization` to take just the org id, this will change to just `action.value`.